Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1284,6 +1284,26 @@ static HTMLElement* topmostPopoverAncestor(Element& newPopover)
     return topmostAncestor.get();
 }
 
+void HTMLElement::checkAndPossiblyClosePopoverStack()
+{
+    if (!document().settings().popoverAttributeEnabled() || !document().hasTopLayerElement())
+        return;
+
+    Vector<Element*> topLayerPositions;
```

Comment:
Please don't store raw pointer in a container. Probably want to use WeakPtr or RefPtr here, depending on whether or not you'd like to keep the elements alive.